Configure.cmake

```diff
@@ -136,6 +142,9 @@ if(CMAKE_C_COMPILER_ID MATCHES "(GNU|Clang)")
 
   # Warning flags.
   set(FLAGS_WALL "-Wall -Wno-unused -Wno-attributes")
+  if(CMAKE_C_COMPILER_ID MATCHES "GNU")
+    string(CONCAT FLAGS_WALL ${FLAGS_WALL} " -Wno-psabi")
```

Comment:
It would be good to have a comment saying why this option is needed to compile libdft.